### PR TITLE
feat(bigtable): Incrementable values

### PR DESCRIPTION
**Pull request by AlisskaPie**

### CODE REVIEW

**Comment by billyjacobson** [comment]
Looks good from a Bigtable perspective

**Comment by billyjacobson** [comment]
@tbpg This looks good on my end, could you give it a language review? Thanks

File: bigtable/bigtable.go [hunk 1]
```diff
@@ -619,6 +621,14 @@ func (m *Mutation) Set(family, column string, ts Timestamp, value []byte) {
 	}}})
 }
 
+// SetInt64 sets a value in a specified column, with the given timestamp.
+// It performs the 64-bit big-endian signed integer encoding.
+func (m *Mutation) SetInt64(family, column string, ts Timestamp, value int) {
```
Comment (tbpg) [inline bigtable/bigtable.go:626]: Seems like this should be an `int64` argument if we always treat it as an `int64`, especially given the name of the method?

File: bigtable/bigtable.go [hunk 2]
```diff
@@ -619,6 +621,14 @@ func (m *Mutation) Set(family, column string, ts Timestamp, value []byte) {
...
+	buf := new(bytes.Buffer)
```
Comment (tbpg) [inline bigtable/bigtable.go:627]:
```suggestion
	var buf bytes.Buffer
```

File: bigtable/bigtable.go [hunk 3]
```diff
@@ -619,6 +621,14 @@ func (m *Mutation) Set(family, column string, ts Timestamp, value []byte) {
...
+	binary.Write(buf, binary.BigEndian, int64(value))
```
Comment (tbpg) [inline bigtable/bigtable.go:628]: `Write` returns an error -- we should check it.

File: bigtable/filter.go [hunk 4]
```diff
@@ -125,6 +127,15 @@ func (cf columnFilter) proto() *btpb.RowFilter {
 // See https://github.com/google/re2/wiki/Syntax for the accepted syntax.
 func ValueFilter(pattern string) Filter { return valueFilter(pattern) }
 
+// ValueFilterInt64 returns a filter that matches cells whose value
+// matches the provided RE2 pattern.
+// It performs the 64-bit big-endian signed integer encoding.
+func ValueFilterInt64(pattern int) Filter {
```
Comment (tbpg) [inline bigtable/filter.go:133]: Same `int64` argument and `binary.Write` comments here and for `ValueRangeFilterInt64`.

File: bigtable/bigtable.go (hunk 1 above)
Comment (tbpg) [inline bigtable/bigtable.go:626]: Looks like we need tests for this.

**Review by tbpg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**